assemblies/installing-odh-v2.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: ASSEMBLY
+//pv2hash: 
+
+ifdef::context[:parent-context: {context}]
+
+:context: installv2
+
+[id="installing-odh-v2_{context}"]
+= Installing Open Data Hub (v2)
+
+[role='_abstract']
+You can install the Open Data Hub Operator version 2 on your OpenShift Container Platform from the OpenShift web console. 
```

Comment:
Similar comment to https://github.com/opendatahub-io/opendatahub-documentation/pull/55/files#r1386903155.